frontend/packages/core/src/Input/select.tsx

```diff
@@ -74,22 +177,20 @@ const Select: React.FC<SelectProps> = ({
   }, []);
 
   return (
-    <FormControl key={name} data-max-width={maxWidth}>
-      <InputLabel shrink color="secondary">
-        {label}
-      </InputLabel>
+    <StyledFormControl key={name} fullWidth disabled={disabled} error={error}>
+      {label && <StyledInputLabel>{label}</StyledInputLabel>}
       <StyledSelect
         value={options[selectedIdx]?.value || options[selectedIdx].label}
         onChange={updateSelectedOption}
-        fullWidth
       >
         {options.map(option => (
           <MenuItem key={option.label} value={option?.value || option.label}>
             {option.label}
           </MenuItem>
         ))}
       </StyledSelect>
-    </FormControl>
+      {helperText && <StyledFormHelperText>{error && <ErrorIcon />}{helperText}</StyledFormHelperText>}
```

Comment:
I think you'll probably carry over the fix from the text field but just so I don't forget the error icon should be centered here.